### PR TITLE
Add README.md + README.en.md

### PR DESCRIPTION
**Pull request by raindropdrop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,46 @@
...
-* https://github.com/AgoraIO/OpenLive-Voice-Only-Android/issues
+- 加入通话和离开通话；
+- 主播和观众模式切换；
+- 静音和解除静音；
+- 切换扬声器和听筒；
 
+你可以在这里查看入门版的示例项目：[Agora-Android-Voice-Tutorial-1to1](https://github.com/AgoraIO/Agora-Android-Voice-Tutorial-1to1) 
 
-## Build Instructions
+可以在这里查看进阶示例项目：[OpenLive-Voice-Only-Android](https://github.com/AgoraIO/OpenLive-Voice-Only-Android/tree/dev/1.12.0)
```
Comment (guohai) [inline README.md]: no need this line, this is already the full-fledged sample for voice only broadcasting. Maybe you can add the link of video sample(OpenLive-Android) here, but you need to make this change for the other projects also.

File: README.en.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+With this sample app, you can:
+
+- Join / leave channel
+- Set role as broadcaster or audience
+- Mute / unmute audio
+- Switch speaker
+
+A tutorial demo can be found here: [Agora-Android-Voice-Tutorial-1to1](https://github.com/AgoraIO/Agora-Android-Voice-Tutorial-1to1)
+
+Further tutorial demo can be found here: [OpenLive-Voice-Only-Android](https://github.com/AgoraIO/OpenLive-Voice-Only-Android/tree/dev/1.12.0)
```
Comment (guohai) [inline README.en.md]: same as the EN version

File: README.md (hunk 1 above)
Comment (raindropdrop) [inline README.md]: fixed

File: README.en.md (hunk 2 above)
Comment (raindropdrop) [inline README.en.md]: fixed

**Comment by raindropdrop** [comment]
Just fixed all problems and pushed new files. Waiting for further review.

**Pull request merged**